docs/source/guide/deployment/guide/docker.rst

```diff
@@ -0,0 +1,223 @@
+======
+Docker
+======
+
+.. Warning::
+  There are limitations to scaling this deployment strategy, as all nodes are
+  running in the same host.
+
+  This deployment is recommended for Tests, Development, POC and POV.
+
+.. Note::
+    * For this deployment we will use RabbitMQ as the
+      :ref:`message queue/broker <guide/deployment/planning/deployment:Message Queue/Broker: Redis or RabbitMQ server>`.
+
+    * This deployment uses the built-in RSTUF Authentication/Authorization
+
+
+The All-in-one host deployment Repository Service for TUF (RSTUF) uses Docker
+Engine and Docker Compose yaml file to deploy RSTUF and all the services as
+containers.
+
+.. note::
+  See the complete :ref:`guide/deployment/planning/deployment:Deployment Configuration`
+  for in-depth details.
+
+Requirements
+============
+
+Software and tools
+------------------
+
+  * Docker Engine (with docker-compose)
+  * Python >= 3.10
+  * pip
+
+
+Online Key
+----------
+
+This deployment requires the :ref:`guide/general/keys:Online Key`.
+See the chapter :ref:`guide/general/keys:Singing Keys`
+
+.. centered:: You can skip it if you already have the online key.
```

Comment:
```suggestion
.. centered:: Skip this section if an online key has already been secured.
```

Reply:
I'm doing a small change: `Skip this section if an online key has already been generated.`